src/applications/hca/utils/actions/disability-rating.js

```diff
@@ -0,0 +1,55 @@
+import recordEvent from '~/platform/monitoring/record-event';
+import { apiRequest } from '~/platform/utilities/api';
+
+import { DISABILITY_PREFIX, DISABILITY_RATING_ACTIONS } from '../constants';
+import {
+  isClientError,
+  isServerError,
+  parseResponseErrors,
+} from '../helpers/disability-rating';
+
+/**
+ * Action to fetch users total disability rating
+ * @returns {Promise} - resolves to calling the reducer to set the correct state variables
+ * for disability rating
+ */
+export function fetchTotalDisabilityRating() {
+  return dispatch => {
+    const {
+      FETCH_DISABILITY_RATING_STARTED,
+      FETCH_DISABILITY_RATING_FAILED,
+      FETCH_DISABILITY_RATING_SUCCEEDED,
+    } = DISABILITY_RATING_ACTIONS;
+    const requestUrl = `/health_care_applications/rating_info`;
+
+    dispatch({ type: FETCH_DISABILITY_RATING_STARTED });
+
+    return apiRequest(requestUrl)
+      .then(result => {
+        recordEvent({ event: `${DISABILITY_PREFIX}-combined-load-success` });
+        const {
+          data: { attributes: response },
+        } = result;
+        return dispatch({ type: FETCH_DISABILITY_RATING_SUCCEEDED, response });
+      })
+      .catch(result => {
+        const { code, detail } = parseResponseErrors(result);
```

Comment:
Good catch here. I'm not sure anything actually gets returned outside of a 2xx, 4xx or 5xx. BUT... just returning `null`, as that function does when none of the cases are truthy, would break this line. I have updated the default return, for the moment, to be an object with both `code` and `detail` as `null`. We can do some additional discovery on the endpoint responses to see if there are any other possibilities.